tiny_gltf.h

```diff
@@ -1536,13 +1538,21 @@ class TinyGLTF {
 #ifndef TINYGLTF_USE_RAPIDJSON
 #include "json.hpp"
 #else
+#ifdef TINYGLTF_USE_HUNTER
```

Comment:
Disable the `TINYGLTF_IMPLEMENTATION` define if HUNTER_ENABLED and create a `cpp` file that builds tinygltf for you. This avoids conflicts with downstream projects having different implementations built.